jekyll/_cci2/contexts.md

```diff
@@ -32,21 +33,9 @@ workflows:
             context: org-global
 ```
 
-## Environment Variables in Contexts
-Contexts have a set of environment variables defined as name/value pairs. When a job is run with a Context the environment variables of the Context are injected at runtime. The order of precedence is important to remember when working with environment variables that might overlap in name. Following is the order, with later injection always taking precedence when the steps of your job run:
+**Note:** Environment variables are considered in a specific order, as follows:
 1. Project-level environment variables set on the Project Settings page.
 2. Context environment variables (assuming the user has access to the Context).
-3. Environment variables set in your config YAML with the `environment` key of a job definition.
+3. Environment variables set in the `config.yml` file with the `environment` key of a job definition.
 4. Environment variables declared inside a shell command in a `run` step of a job.
```

Comment:
The (existing) wording here is ambiguous. There are two was a user could specify an env var in a `run` step:
- inside the shell command like `FOO=bar make install`
- in the `environment` Key of a `run` step.